docker/Dockerfile

```diff
@@ -14,14 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu
+#FROM ubuntu
+FROM ubuntu:16.04
```

Comment:
Why lock this down to an older version of ubuntu?

Reply:
That's a mistake, leftover from testing on 16.04. I've pushed the change and payed attention, this time.